### PR TITLE
feat: Move recording filters into side

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Filters for Recordings sticks out in an odd way. Before solving it more holistically we decided to move it to the side when the screen is big enough to make it more manageable

## Changes

* Moves filters to the side
* Makes it togglable so it can be hidden and shown by preference

### Playlist view wide
Screenshot 2022-09-16 at 18 09 15

### Playlist view small
Screenshot 2022-09-16 at 18 09 26

### Non-Playlist view
Screenshot 2022-09-16 at 18 12 13

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by rcmarron — APPROVED** [review]
Looks great! Definitely an improvement! 

As I use it, there's some odd behavior where the filters can start to push the recording list itself below the player. See this example with 2 filters:
image

Definitely not a blocker, but I think we can iterate on this design as we go - maybe the properties can be tightened up like we talked about in standup.

**Comment by benjackwhite** [comment]
> Definitely not a blocker, but I think we can iterate on this design as we go - maybe the properties can be tightened up like we talked about in standup.

Yeah this is definitely "on purpose" at the moment but I agree it ends up making the list harder to see. Perhaps we need to scroll all the way to the top of the page to see the "hide filters" button again or something...

File: frontend/src/lib/components/PropertyKeyInfo.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
     align-items: center;
     vertical-align: middle;
     overflow: hidden;
+    max-width: 100%;
 
     .PropertyKeyInfo__text {
         max-width: 400;
```
Comment (alexkim205) [inline frontend/src/lib/components/PropertyKeyInfo.scss:11]: Unrelated to this PR but I think this is a typo.

File: frontend/src/scenes/session-recordings/sessionRecordingsTableLogic.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export const sessionRecordingsTableLogic = kea<sessionRecordingsTableLogicType>(
         },
         loadNext: true,
         loadPrev: true,
-        enableFilter: true,
+        setFiltersEnabled: (showing: boolean) => ({ showing }),
```
Comment (alexkim205) [inline frontend/src/scenes/session-recordings/sessionRecordingsTableLogic.ts:87]: nice, love that we can hide stuff now

**Review by alexkim205 — APPROVED** [review]
lgtm

File: frontend/src/lib/components/PropertyKeyInfo.scss (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/lib/components/PropertyKeyInfo.scss:11]: Good catch, that might have been me some other time 🤦  I'm pretty sure it is doing nothing so I'll remove it

**Pull request merged**